packages/overlay/overlay-trigger.md

```diff
@@ -1,16 +1,6 @@
-## Description
```

Comment:
Moving the “Description” heading down denormalizes this content from other pages in the docs.